client/lib/lib.rs

```diff
@@ -167,3 +176,436 @@ pub fn get_block(
 ) -> Result<JsonRpc> {
     RpcCall::new(maybe_rpc_id, node_address, verbose)?.get_block(maybe_block_hash)
 }
+
+/// Puts a deploy on a node.
+///
+/// * `maybe_rpc_id` is used for RPC-ID as required by the JSON-RPC specification, and is returned
+///   by the node in the corresponding response.  It must be either able to be parsed as a `u32` or
+///   empty.  If empty, a random ID will be assigned.
+/// * `node_address` identifies the network address of the target node's HTTP server, e.g.
+///   `"http://127.0.0.1:7777"`.
+/// * When `verbose` is `true`, the request will be printed to `stdout`.
+/// * `deploy` - Deploy arguments container struct holding options for this deploy. See
+///   `DeployStrParams`.
+/// * `session` - Session arguments container struct holding options for this deploy. See
+///   `SessionStrParams`.
+/// * `payment` - Payment arguments container struct holding options for this deploy. See
+///   `PaymentStrParams`.
```

Comment:
I think we should keep a consistent style: either have all parameters described in the form of `* param - description` or use full sentences.  I don't really mind which, but I think we should be consistent.  This applies to a few of the doc comments added in this commit.

Reply:
Got it, thanks.